src/status_im/common/standard_authentication/standard_auth/slide_button/view.cljs

```diff
@@ -26,14 +26,18 @@
                  size
                  theme
                  blur?
+                 on-close
                  container-style]
-          :or   {container-style {:flex 1}}}]
+          :or   {container-style {:flex 1}
+                 on-close        #()}}]
```

Comment:
Just a reminder that if a caller of `view-internal` pass `on-close` as nil for any reason (easy to happen with Clojure conditionals), then the destructring code to set a default `on-close` won't be used and the app will break when `:on-complete` runs because `on-close` is called unchecked.

The pattern `(when f (f))` is more robust here, and the default destructuring can be removed.

Reply:
It's safer to do: `(when (fn? f) (f))`, no?

Reply:
I like @J-Son89 's approach! too expressive and cheap to implement

Reply:
> It's safer to do: (when (fn? f) (f)), no?

Definitely safer, but I don't remember a case where we passed a string or other primitive instead of a function. As far as I've seen, it's either the value exists and it's a valid function instance or it's nil.